dependabot/resources/extensions.json

```diff
@@ -655,6 +655,7 @@
                 "module-ballerinax-mssql",
```

Comment:
All of these are autogenerated files, only add the module in `dependabot/resources/modules.json`

Reply:
changed in `dependabot/resources/module_list.json` in https://github.com/ballerina-platform/ballerina-release/pull/1110/commits/5c9614c511a47fd7c8aa5a3c5214fd3d12ec4811